style.css

```diff
@@ -0,0 +1,48 @@
+/* {
+  border: 1px dashed black;
+}*/
+
+* {
+  font-family: Verdana, Geneva, sans-serif;
+}
+
+button {
+  background-color: #4CAF50;
+  border: none;
+  width: 50%;
+  text-align: center;
+  text-decoration: none;
+  font-size: 1em;
+  border-radius: 4px;
+}
+
+button:hover {
+  background-color: #3e8e41
+}
+
+input {
+  width: 100%;
+}
+
+/*Does not display until successful login*/
+#welcomeScreen {
+  display: none;
+}
+
+/*The input boxes and button are smaller when screen is bigger*/
+@media (min-width: 401px) {
+  .largeView {
```

Comment:
I don't know if you need this class? You shouldn't need to change the class on the header.